src/Notes/WC_Admin_Notes_Facebook_Marketing_Expert.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * WooCommerce Admin Facebook Marketing Expert Note Provider.
+ *
+ * Adds notes to the merchant's inbox concerning Facebook Marketing Expert.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Facebook_Marketing_Expert
+ */
+class WC_Admin_Notes_Facebook_Marketing_Expert {
+
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-facebook-marketing-expert';
+
+	/**
+	 * Name of plugin file.
+	 */
+	const PLUGIN_FILE = 'facebook-for-woocommerce/facebook-for-woocommerce.php';
+
+	/**
+	 * Possibly add note.
+	 */
+	public static function possibly_add_note() {
+
+		// Check if the note can and should be added.
+		if ( ! self::can_be_added() ) {
+			return;
+		}
+
+		// Only add the note to stores with Facebook for WooCommerce installed.
+		if ( ! self::is_facebook_for_woocommerce_installed() ) {
+			return;
+		}
+
+		// Only add the note to stores with at least 30 orders in the last month.
+		if ( ! self::orders_last_month() >= 30 ) {
```

Comment:
This condition doesn't work for me. I'm seeing the notice with just 1 order in the last month. 

I'm guessing it's due to Operator Precedence where the negation (`!`) takes precedence. Perhaps use:
```php
self::orders_last_month() < 30
```